src/libsystemd/sd-bus/bus-message.c

```diff
@@ -2961,6 +2981,10 @@ int bus_message_seal(sd_bus_message *m, uint64_t cookie, usec_t timeout) {
         return 0;
 }
 
+_public_ int sd_bus_message_seal(sd_bus_message *m, uint64_t cookie, uint64_t timeout_usec) {
+        return bus_message_seal(m, cookie, timeout_usec);
```

Comment:
if these two functions are identical, then one should not be there. i.e. please simply rename bus_message_seal to sd_ bus_message_seal(). And while doing that make sure, you change any assert() in there to assert_return(), as the latter is what we do for interfaces for external code